<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WPGraphAnalysisVisualWidget</class>
 <widget class="QWidget" name="WPGraphAnalysisVisualWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QFrame" name="frZ">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QFrame" name="frZTitle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="lblZTitle">
            <property name="text">
             <string>Общая масса</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frZSignal">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="AreaGraph" name="wgtAreaZ" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frY">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QFrame" name="frYTitle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="lblYTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Смещение вдоль туловища</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frYSignal">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="AreaGraph" name="wgtAreaY" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QScrollBar" name="sbSignal">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frTools">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frScale">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="lblScale">
           <property name="text">
            <string>Масштаб</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbScale">
           <property name="maxVisibleItems">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFullSignal">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../../kernel.qrc">
          <normaloff>:/images/Tools/FullSignal.png</normaloff>:/images/Tools/FullSignal.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frFullSignal">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnHScalePlus">
           <property name="maximumSize">
            <size>
             <width>13</width>
             <height>13</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../../kernel.qrc">
             <normaloff>:/images/Tools/Plus_10.png</normaloff>:/images/Tools/Plus_10.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>9</width>
             <height>9</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnHScaleMinus">
           <property name="maximumSize">
            <size>
             <width>13</width>
             <height>13</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../../kernel.qrc">
             <normaloff>:/images/Tools/Minus_10.png</normaloff>:/images/Tools/Minus_10.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnExport">
        <property name="text">
         <string>Экспорт сигналов</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnApnoe">
        <property name="text">
         <string>Расчет показателей...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>713</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AreaGraph</class>
   <extends>QWidget</extends>
   <header location="global">areagraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../kernel.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbScale</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>WPGraphAnalysisVisualWidget</receiver>
   <slot>scaleChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>914</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFullSignal</sender>
   <signal>clicked(bool)</signal>
   <receiver>WPGraphAnalysisVisualWidget</receiver>
   <slot>btnFulSignalClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>914</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnHScalePlus</sender>
   <signal>clicked()</signal>
   <receiver>WPGraphAnalysisVisualWidget</receiver>
   <slot>btnPlusClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>915</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnHScaleMinus</sender>
   <signal>clicked(bool)</signal>
   <receiver>WPGraphAnalysisVisualWidget</receiver>
   <slot>btnMinusClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>576</y>
    </hint>
    <hint type="destinationlabel">
     <x>915</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbSignal</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WPGraphAnalysisVisualWidget</receiver>
   <slot>signalScroll(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>914</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnExport</sender>
   <signal>clicked()</signal>
   <receiver>WPGraphAnalysisVisualWidget</receiver>
   <slot>on_signalExport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>914</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnApnoe</sender>
   <signal>clicked()</signal>
   <receiver>WPGraphAnalysisVisualWidget</receiver>
   <slot>on_apnoeFactorsCalculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>914</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>scaleChange(int)</slot>
  <slot>btnFulSignalClicked(bool)</slot>
  <slot>btnPlusClicked()</slot>
  <slot>btnMinusClicked()</slot>
  <slot>signalScroll(int)</slot>
  <slot>on_signalExport()</slot>
  <slot>on_apnoeFactorsCalculate()</slot>
 </slots>
</ui>
